pyquil/gate_matrices.py

```diff
@@ -181,6 +184,13 @@ def PSWAP(phi):
                      [0, 0, 0, 1]])
 
 
+def XY(phi):
```

Comment:
should we add/rename XY in `gates.py` as well?

Reply:
I'll make a separate PR for it

Reply:
I already included it in this one.